### PR TITLE
Add rack-cors gem and configure initializer file for localhost access

### PR DESCRIPTION
**Pull request by pcbennett108**
#### Updates Made -
- Type of change: Install GEM
- Database/Migrations: n/a
- Routes: n/a
- Models: n/a
- Controllers: n/a
- API/JSON: n/a
- Testing:
  - Models: Tests pass
  - Features: Tests pass
- Considerations(Extra notes, refactors, etc.): Uncommented rack-core gem in gem file. Added base configuration to config/initializers/cors.rb. I only added access from localhost 3000. The deployed frontend address will need to be added once it is deployed. I restricted actions to GET for now.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -162,6 +162,8 @@ GEM
       nio4r (~> 2.0)
```
Comment (kameronk92) [inline Gemfile.lock:162]: I think we can add Gemfile.lock to our gitignore but ok for now

File: config/initializers/cors.rb [hunk 2]
```diff
@@ -5,12 +5,15 @@
...
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "http://localhost:3000",
+            "http://127.0.0.1:3000"
+
+            # "http://yourwebsite.production.app" frontend deploy URL
```
Comment (kameronk92) [inline config/initializers/cors.rb:14]: Do we need to add our production url here?

Comment (pcbennett108) [reply config/initializers/cors.rb:14]: I added it just to be safe.

**Comment by Jesuitman** [comment]
Awesome good work finding this Paul!

**Pull request merged**